src/Access/Authentication.h

```diff
@@ -88,8 +88,8 @@ class Authentication
     void setServerName(const String & server_name_);
 
     /// Checks if the provided password is correct. Returns false if not.
-    /// User name and external authenticators' info are used only by some specific authentication type (e.g., LDAP_SERVER).
-    bool isCorrectPassword(const String & password_, const String & user_, const ExternalAuthenticators & external_authenticators) const;
+    /// User name and external authenticators are used by the specific authentication types only (e.g., LDAP_SERVER).
```

Comment:
This comment is one of the reasons why `password_` used to be passed first: it's used almost for each authentication type unlike `user_`.